profile/src/main/kotlin/com/imashnake/animite/profile/tabs/Anime.kt

```diff
@@ -48,7 +67,16 @@ private fun UserMediaList(
                 MediaSmall(
                     image = media.coverImage,
                     label = media.title,
-                    onClick = {},
+                    onClick = {
+                        onNavigateToMediaItem(
+                            MediaPage(
+                                id = media.id,
+                                // TODO: Unhardcode these and make them unique.
+                                source = media.title.orEmpty(),
+                                mediaType = MediaType.ANIME.rawValue,
```

Comment:
if we're going back to storing these results, are these fields something we will have saved, so we'll only need to navigate by ID?

Reply:
Can't navigate by id only anymore because of shared element transitions, i.e., we can have the same media card in two different lists and the shared element key has to be unique for each so we need to encode the list type and destinations in the key (see `SharedContentKey.toString()`).